backend/src/bundles/resumes/resume.controller.ts

```diff
@@ -1,28 +1,86 @@
-import { ApiPath, HttpCode } from 'shared/build/index.js';
+import { HttpError, ResumesApiPath } from 'shared/build';
 
 import {
     type ApiHandlerOptions,
     type ApiHandlerResponse,
-} from '~/common/controller/controller.js';
-import { Controller } from '~/common/controller/controller.package.js';
-import { type ILogger } from '~/common/logger/logger.js';
+} from '~/common/controller/controller';
+import { Controller } from '~/common/controller/controller';
+import { ApiPath } from '~/common/enums/enums';
+import { HttpCode } from '~/common/http/http';
+import { type ILogger } from '~/common/logger/logger';
 
-import { ResumeApiPath } from './enums/enums.js';
+import { type IResumeService } from './interfaces/interfaces.js';
 import {
+    type Resume,
     type ResumeAiScoreRequestDto,
     type ResumeAiScoreResponseDto,
-    type ResumeService as TResumeService,
+    type ResumeCreateItemRequestDto,
+    type ResumeGetAllResponseDto,
+    type ResumeUpdateItemRequestDto,
 } from './types/types.js';
 
 class ResumeController extends Controller {
-    private resumeService: TResumeService;
+    private resumeService: IResumeService<Resume>;
 
-    public constructor(logger: ILogger, resumeService: TResumeService) {
+    public constructor(logger: ILogger, resumeService: IResumeService<Resume>) {
         super(logger, ApiPath.RESUMES);
+
         this.resumeService = resumeService;
 
         this.addRoute({
-            path: ResumeApiPath.SCORE,
+            path: ResumesApiPath.ROOT,
+            method: 'POST',
+            handler: (options) =>
+                this.create(
+                    options as ApiHandlerOptions<{
+                        body: ResumeCreateItemRequestDto;
+                    }>,
+                ),
+        });
+        this.addRoute({
+            path: ResumesApiPath.ID,
+            method: 'GET',
+            handler: (options) =>
+                this.findById(
+                    options as ApiHandlerOptions<{ params: { id: string } }>,
```

Comment:
Create type for `{ id: string }` in `shared` package and use it here. Check other files

Reply:
I replaced `{ id: string }` to `IdParameter` type.